container_files/assets/www/doc/builtin/functions/Kmeans.md

```diff
@@ -49,7 +49,7 @@ Undecided | 0.5 | 0.5
 
 Which represents the following situation:
 
-![Sliced Dataset](/doc/builtin/img/KMeansFunction.svg)
+![Sliced Dataset](../img/KMeansFunction.svg)
```

Comment:
this is inconsistent with the change above, in terms of their relative positions. why `../img` in one case but `img/` in the other?